remappings.txt

```diff
@@ -1,7 +1,6 @@
 solmate/=lib/solmate/
 permit2/=lib/permit2/
 forge-std/=lib/forge-std/src/
-@openzeppelin/contracts=node_modules/@openzeppelin/contracts
 @uniswap/v3-core/=node_modules/@uniswap/v3-core/
```

Comment:
for some reason, foundry was auto-remapping `@openzeppelin/contracts/=lib/v4-periphery/lib/v4-core/lib/openzeppelin-contracts/` which totally broke contract verification

poked around, and it seems to be implicitly be using v4-periphery's remappings 🤔